samples/micrometer-samples-spring-framework6/build.gradle

```diff
@@ -10,7 +10,6 @@ if (!javaLanguageVersion.canCompileOrRun(17)) {
 
 dependencies {
     implementation project(":micrometer-core")
-    implementation project(":micrometer-observation")
```

Comment:
Does this have anything to do with your issue in the root `build.gradle`?

Reply:
Yes. There was an issue with root `build.gradle`.
But, this is just optimization that 'micrometer-core' already has 'micrometer-observation' with `api`.
So, this line is unnecessary